server/scrapping/scrapingArabicLyrics.js

```diff
@@ -0,0 +1,36 @@
+import puppeteer from "puppeteer"
```

Comment:
why is this a part of the PR?

Reply:
This is a separate PR - see PR 69 scraping arabic lyrics